en/sensor/thunderfly_tachometer.md

```diff
@@ -70,15 +70,24 @@ When using CUAV V5+ or CUAV Nano:
 
 
 ### Function validation
-You can verify functionality in multiple ways:
+You can verify functionality in multiple ways.
+
+
+#### Pomocí PX4 (NuttX) konzole
```

Comment:
Note, this is not English.

Reply:
Yes, sorry for that.  Corrected and now ready to review.